.scripts/run_config.sh

```diff
@@ -98,7 +107,7 @@ do
 
   # If printing sql to file, comment out metadata strings in the SQL
   if [ ! -z "$OUTPUT_PATH" ]; then
-    sed -E -i '' '/^([0-9]{4}\/(0[1-9]|1[0-2])\/(0[1-9]|[1-2][0-9]|3[0-1]) (2[0-3]|[01][0-9]):[0-5][0-9]:[0-5][0-9] Step name:|Query name: |Query path: )/ s/^/-- /' $OUTPUT_FILE
+    sed -E -i '/^([0-9]{4}\/(0[1-9]|1[0-2])\/(0[1-9]|[1-2][0-9]|3[0-1]) (2[0-3]|[01][0-9]):[0-5][0-9]:[0-5][0-9] Step name:|Query name: |Query path: )/ s/^/-- /' $OUTPUT_FILE
```

Comment:
This change also breaks the intended behaviour - the intention of this line is to comment out the metadata that sql-runner produces with `fillTemplates` (below). Without the extra `''` it doesn't do this.

```
-- 2021/02/05 19:13:29 Step name: 01-page-views-join-setup
-- Query name: 01-page-views-join-setup
-- Query path: /Users/cog/GitHub/data-models/web/v1/bigquery/sql-runner/sql/custom/02-page-views-join/01-page-views-join-setup.sql
```

Reply:
Ok, for me it is the opposite happening. 
```
sed: can't read /^([0-9]{4}\/(0[1-9]|1[0-2])\/(0[1-9]|[1-2][0-9]|3[0-1]) (2[0-3]|[01][0-9]):[0-5][0-9]:[0-5][0-9] Step name:|Query name: |Query path: )/ s/^/-- /: No such file or directory
```

`sed (GNU sed) 4.7
Packaged by Debian`

Reply:
You're not alone...
https://unix.stackexchange.com/questions/92895/how-can-i-achieve-portability-with-sed-i-in-place-editing

Using `perl` instead or not using `-i` seems like the best option.

Reply:
Ah it all makes sense now. OK I'm cool with either approach.

Reply:
I see this has changed to `perl` I'll give it a test, but if it doesn't play nicely with Mac I can fix it in the BQ release - no need to block this release for it.